### PR TITLE
Fix shellcheck warnings/errors in cluster/gce/gci/master-helper.sh

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Fix shellcheck failures in cluster/gce/gci/master-helper.sh.

**Which issue(s) this PR fixes**:

Ref #72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
none
```

### CODE REVIEW

File: cluster/gce/gci/master-helper.sh [hunk 1]
```diff
@@ -142,9 +145,11 @@ function create-master-instance-internal() {
     enable_ip_aliases=false
   fi
 
-  local network=$(make-gcloud-network-argument \
+  # shellcheck disable=SC2153 # 'NETWORK' is assigned by upstream
+  network=$(make-gcloud-network-argument \
     "${NETWORK_PROJECT}" "${REGION}" "${NETWORK}" "${SUBNETWORK:-}" \
     "${address:-}" "${enable_ip_aliases:-}" "${IP_ALIAS_SIZE:-}")
+  local network
```
Comment (sftim) [inline cluster/gce/gci/master-helper.sh]: I'd expect `local network` to come before the assignment to `$network`. Did I miss a detail?

Comment (gavinfish) [reply cluster/gce/gci/master-helper.sh]: Thanks @sftim! fixed

**Comment by joakimr-axis** [comment]
Looks nice!

**Pull request closed without merging**